www/source/demo/windows/steps/7.html.slim

```diff
@@ -14,6 +14,8 @@ section
      Set-Location {{pkg.svc_path}}
      if(Test-Path var) { Remove-Item var -Recurse -Force }
      New-Item -Name var -ItemType Junction -target "{{pkg.path}}/www" | Out-Null
+     Set-Location {{pkg.svc_path}}\var
+     New-Item -Name Web.config -ItemType SymbolicLink -target "{{pkg.svc_config_path}}/Web.config" -Force | Out-Null
 
```

Comment:
same as above.

Reply:
Yeah we run into these issues following steps on the demo that the `web.config` wasn't found when starting the application in the docker container. If you get this working as it is today let me know.

Reply:
Hmm you know what the confusion is. This step: https://www.habitat.sh/demo/windows/steps/8/#export-container suggests you should start your app before you do the work in step 9, so at this point your application will not start up. That's where some people running through the demo got stuck.

Reply:
Ah I see what happened here. At some point we changed the perms on the `config` directory and needed to add:
```
# The svc_config_path lacks an ACL for the USERS group
# so we need to ensure the app pool user can access those files
$pool = "{{cfg.app_pool}}"
$access = New-Object System.Security.AccessControl.FileSystemAccessRule `
  "IIS APPPOOL\$pool",`
  "ReadAndExecute",`
  "Allow"
$acl = Get-Acl "{{pkg.svc_config_path}}/Web.config"
$acl.SetAccessRule($access)
$acl | Set-Acl "{{pkg.svc_config_path}}/Web.config"
```
to the `run` hook. In the tutorial, we should have added that in step 9 and omitted it from step 7.

I'll can add that to this PR.